### DISCUSSION TITLE
/articles/build-web-apps-not-microsoft-365-apps/

### DISCUSSION THREAD

**Discussion by giscus** [Voitanos/voitanos.io-comments#315] [Comments]
Don’t get tunnel vision: Expand beyond Microsoft Teams and SharePoint apps. Prioritize web apps for a more versatile Microsoft 365 development approach.
https://www.andrewconnell.com/articles/build-web-apps-not-microsoft-365-apps/

**Comment by osjariri** [comment]
Thanks Andrew, I was always thinking the same, why would we stuck our our code with limited cabapilities and limited versions of libraries (TS, React,...), and spend hard time follow the versions, why we not just crreate a webapp and host it in azure for example, and add a link in SP online.
